src/style.css

```diff
@@ -102,11 +103,81 @@ button {
   border: 1px solid #888;
   padding: 0.5rem 0.9rem;
   border-radius: 4px;
+  position: relative;
+}
+
+.cart-badge {
+  position: absolute;
+  top: -9px;
+  right: -14px;
+  transform: translateY(-4%);
+  background-color: #1877f2;
+  color: white;
+  font-size: 0.8rem;
+  font-weight: bold;
+  padding: 3px 9px;
+  border-radius: 2px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
 }
 
 @media (min-width: 768px) {
   .button-container {
-    display: block;
+    display: flex;
+  }
+
+  .sign-in-btn {
```

Comment:
since the color background color, etc properties will be sam e in all screen sizes. you can add them in default styles. no need to use media query. media query is for the changes in different screen size